real_check/real_check.md

```diff
@@ -0,0 +1,10 @@
+# real_check 
+
+This tool checks if the given library contains a pair of symbols of the form `[symbol]` and `real_[symbol]`. If the given library contains no such pair, the tool returns exit code `0`. If the given library contains such a pair, the tool prints these symbols to stderror and returns exit code `1`. If the tool is unable to determine if the library contains such a pair, it returns exit code `2`.
```

Comment:
>urns exit co [](http://example.com/codeflow?start=277&length=12)

Can we split the text over a couple of lines. Usually one sentence per line makes it easy enough to look at.

Reply:
Done.